server/base/src/main/java/org/apache/accumulo/server/tables/TableManager.java

```diff
@@ -260,7 +252,7 @@ public void accept(WatchedEvent event) {
       final String zPath = event.getPath();
```

Comment:
I have the same question here about the Watcher even path being full or relative.

Reply:
I created an IT in 602fd4df7591971362373538df5c636d39b20bbc to verify the paths are relative. (see earlier comment about the same).